packages/perseus/src/widgets/sorter/sorter.tsx

```diff
@@ -101,21 +96,8 @@ class Sorter extends React.Component<Props, State> {
         this.refs.sortable.moveOptionToIndex(option, index);
     };
 
-    simpleValidate: (arg1: any) => any = (rubric) => {
-        // If this widget hasn't been changed yet, we treat it as "empty" which
-        // prevents the "Check" button from becoming active. We want the user
-        // to make a change before trying to move forward. This makes an
-        // assumption that the initial order isn't the correct order! However,
-        // this should be rare if it happens, and interacting with the list
-        // will enable the button, so they won't be locked out of progressing.
-        if (!this.state.changed) {
-            return {
-                type: "invalid",
-                message: null,
-            };
-        }
-
-        return Sorter.validate(this.getUserInput(), rubric);
+    simpleValidate: (arg1: Rubric) => PerseusScore = (rubric) => {
```

Comment:
No action needed because I addressed this in my other PR, but in the future if you see:

``` TS
myMethod: (genericArgName: SomeArgType) => SomeReturnType = (goodArgName) => {
  return this.whatever;
}
```

I think we could benefit from cleaning it up:

``` TS
myMethod(goodArgName: SomeArgType): SomeReturnType {
  return this.whatever;
}
```

Reply:
Thank you for pointing this out! Does that change it to no longer be an arrow function though and does that matter? I think arrow functions have an effect on code using "this". Looks like arrow functions inherit their "this" from the parent scope ([link](https://www.codementor.io/@dariogarciamoya/understanding-this-in-javascript-with-arrow-functions-gcpjwfyuc)). I'm not really sure if that's relevant here, but since it does use "this," wanted to point it out. It does look _way_ cleaner the second way.